Latest release broke builds

We're now hitting the following error message in our macos builds over in the Bevy repo:

```txt
error[E0425]: cannot find function `clang_CXXMethod_isVirtual` in this scope
    --> /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/bindgen-0.61.0/clang.rs:817:18
     |
817  |           unsafe { clang_CXXMethod_isVirtual(self.x) != 0 }
     |                    ^^^^^^^^^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `clang_CXXMethod_isPureVirtual`
     |
    ::: /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/clang-sys-1.5.1/src/lib.rs:1735:1
     |
1735 | / link! {
1736 | |     pub fn clang_CXCursorSet_contains(set: CXCursorSet, cursor: CXCursor) -> c_uint;
1737 | |     pub fn clang_CXCursorSet_insert(set: CXCursorSet, cursor: CXCursor) -> c_uint;
1738 | |     pub fn clang_CXIndex_getGlobalOptions(index: CXIndex) -> CXGlobalOptFlags;
...    |
2289 | |     pub fn clang_VerbatimLineComment_getText(comment: CXComment) -> CXString;
2290 | | }
     | |_- similarly named function `clang_CXXMethod_isPureVirtual` defined here
```

This appears to be tied to the latest release of `clang-sys`.

https://github.com/bevyengine/bevy/actions/runs/4099201878/jobs/7068888325

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1770,6 +1770,7 @@ link! {
     pub fn clang_CXXMethod_isMoveAssignmentOperator(cursor: CXCursor) -> c_uint;
     pub fn clang_CXXMethod_isPureVirtual(cursor: CXCursor) -> c_uint;
     pub fn clang_CXXMethod_isStatic(cursor: CXCursor) -> c_uint;
+    pub fn clang_CXXMethod_isVirtual(cursor: CXCursor) -> c_uint;
     /// Only available on `libclang` 6.0 and later.
     #[cfg(feature = "clang_6_0")]
     pub fn clang_CXXRecord_isAbstract(cursor: CXCursor) -> c_uint;

